server/routes/imageUploaderAPI.js

```diff
@@ -1,37 +1,54 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
 
-const multer = require('multer');
-const multerS3 = require('multer-s3');
-const AWS = require('aws-sdk');
+const multer = require("multer");
+const multerS3 = require("multer-s3");
+const AWS = require("aws-sdk");
+
+const auth = require("../middleware/auth");
+const db = require("../db/models");
+const mongoose = require("mongoose");
 
 // Set the region
 AWS.config.update({ region: process.env.AWS_REGION });
 
+const BUCKET = process.env.AWS_BUCKET_NAME;
+
 // Create S3 service object
 const s3 = new AWS.S3();
-const FN = Date.now().toString() + '.png'
+const FN = Date.now().toString() + ".png";
```

Comment:
This line is why image does not update!😜🧐

Reply:
Thanks for your help!